django/django_contracts/models/contract.py

```diff
@@ -0,0 +1,80 @@
+from datetime import datetime
+
+from django.db import models, transaction
+from django.db.models import Manager
+from django.urls import reverse
+
+from django_contracts.compat import TimestampMixin
+from django_contracts.models.publishable import PublishableMixin, PublishStatus
+
+
+class LegalContract(TimestampMixin, PublishableMixin):
```

Comment:
Is it intentional to have PublishStatus on both contract and version, or would it suffice to have the value on version?

Reply:
Both are intentional, as that allows the views to work in draft mode if you're an admin user. Draft mode contracts can still be deleted & edited in many ways, whereas published can't.